mbuild/formats/cassandramcf.py

```diff
@@ -1,7 +1,6 @@
 """Cassandra Molecular Connectivity format.
 
-https://cassandra-mc.readthedocs.io/en/latest/guides/input_files.html#
-molecular-connectivity-file
+https://cassandra-mc.readthedocs.io/en/latest/guides/input_files.html #
```

Comment:
This is still incorrect. the `#` takes the user to the subsection in the html page, directly to the MCF file section.
```suggestion
https://cassandra-mc.readthedocs.io/en/latest/guides/input_files.html#molecular-connectivity-file
```

Reply:
fixed